src/network/address.rs

```diff
@@ -258,10 +258,7 @@ impl Encodable for AddrV2Message {
         len += self.time.consensus_encode(w)?;
         len += VarInt(self.services.to_u64()).consensus_encode(w)?;
         len += self.addr.consensus_encode(w)?;
-
-        // consensus_encode always encodes in LE, and we want to encode in BE.
-        //TODO `len += io::Write::write(w, &self.port.to_be_bytes())?;` when MSRV >= 1.32
-        len += self.port.swap_bytes().consensus_encode(w)?;
+        len += io::Write::write(w, &self.port.to_be_bytes())?;
```

Comment:
`write_all`

Reply:
Gets me every time.

Reply:
No worries, maybe it'll become non-issue. See https://github.com/rust-bitcoin/rust-bitcoin/issues/1123#issuecomment-1193158173